config/jobs/kubernetes/sig-node/crio.yaml

```diff
@@ -96,6 +96,38 @@ periodics:
     testgrid-tab-name: ci-crio-cgroupv1-node-e2e-features
     testgrid-alert-email: kubernetes-sig-node-test-failures@googlegroups.com
     description: "OWNER: sig-node; runs NodeFeatures e2e tests with crio master and cgroup v1"
+- name: ci-crio-cgroupv1-node-e2e-eviction
```

Comment:
I understand now that you are trying to add cri-o replacement for docker as well. Would it be better to have the names pattern match between runtimes?

Reply:
between ci-kubernetes-node-kubelet-containerd-eviction and ci-crio-cgroupv1-node-e2e-eviction, can we match the name pattern or cross reference tests?

Reply:
Can we rename in a separate PR? All of the job names are kind of out of sync and I know SIG Testing had previously suggested we take the runtimes out of the job names entirely.

Reply:
https://github.com/kubernetes/test-infra/issues/24641